Description: Use original git-todo-parser instead of fork
 Upstream uses fork. and we use the original version, which is already in
 Debian. There is no API conflicts.
Author: Jongmin Kim <jmkim@debian.org>
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/pkg/app/daemon/rebase.go
+++ b/pkg/app/daemon/rebase.go
@@ -8,7 +8,7 @@
 	"github.com/jesseduffield/lazygit/pkg/common"
 	"github.com/jesseduffield/lazygit/pkg/env"
 	"github.com/jesseduffield/lazygit/pkg/utils"
-	"github.com/stefanhaller/git-todo-parser/todo"
+	"github.com/fsmiamoto/git-todo-parser/todo"
 )
 
 type ChangeTodoAction struct {
--- a/pkg/commands/git_commands/commit_loader.go
+++ b/pkg/commands/git_commands/commit_loader.go
@@ -17,7 +17,7 @@
 	"github.com/jesseduffield/lazygit/pkg/common"
 	"github.com/jesseduffield/lazygit/pkg/utils"
 	"github.com/samber/lo"
-	"github.com/stefanhaller/git-todo-parser/todo"
+	"github.com/fsmiamoto/git-todo-parser/todo"
 )
 
 // context:
--- a/pkg/commands/git_commands/commit_loader_test.go
+++ b/pkg/commands/git_commands/commit_loader_test.go
@@ -12,7 +12,7 @@
 	"github.com/jesseduffield/lazygit/pkg/common"
 	"github.com/jesseduffield/lazygit/pkg/utils"
 	"github.com/samber/lo"
-	"github.com/stefanhaller/git-todo-parser/todo"
+	"github.com/fsmiamoto/git-todo-parser/todo"
 	"github.com/stretchr/testify/assert"
 )
 
--- a/pkg/commands/git_commands/patch.go
+++ b/pkg/commands/git_commands/patch.go
@@ -9,7 +9,7 @@
 	"github.com/jesseduffield/lazygit/pkg/app/daemon"
 	"github.com/jesseduffield/lazygit/pkg/commands/models"
 	"github.com/jesseduffield/lazygit/pkg/commands/patch"
-	"github.com/stefanhaller/git-todo-parser/todo"
+	"github.com/fsmiamoto/git-todo-parser/todo"
 )
 
 type PatchCommands struct {
--- a/pkg/commands/git_commands/rebase.go
+++ b/pkg/commands/git_commands/rebase.go
@@ -11,7 +11,7 @@
 	"github.com/jesseduffield/lazygit/pkg/commands/oscommands"
 	"github.com/jesseduffield/lazygit/pkg/utils"
 	"github.com/samber/lo"
-	"github.com/stefanhaller/git-todo-parser/todo"
+	"github.com/fsmiamoto/git-todo-parser/todo"
 )
 
 type RebaseCommands struct {
--- a/pkg/commands/models/commit.go
+++ b/pkg/commands/models/commit.go
@@ -5,7 +5,7 @@
 
 	"github.com/jesseduffield/lazygit/pkg/utils"
 	"github.com/samber/lo"
-	"github.com/stefanhaller/git-todo-parser/todo"
+	"github.com/fsmiamoto/git-todo-parser/todo"
 )
 
 // Special commit hash for empty tree object
--- a/pkg/gui/controllers/helpers/merge_and_rebase_helper.go
+++ b/pkg/gui/controllers/helpers/merge_and_rebase_helper.go
@@ -13,7 +13,7 @@
 	"github.com/jesseduffield/lazygit/pkg/gui/types"
 	"github.com/jesseduffield/lazygit/pkg/utils"
 	"github.com/samber/lo"
-	"github.com/stefanhaller/git-todo-parser/todo"
+	"github.com/fsmiamoto/git-todo-parser/todo"
 )
 
 type MergeAndRebaseHelper struct {
--- a/pkg/gui/controllers/local_commits_controller.go
+++ b/pkg/gui/controllers/local_commits_controller.go
@@ -15,7 +15,7 @@
 	"github.com/jesseduffield/lazygit/pkg/gui/types"
 	"github.com/jesseduffield/lazygit/pkg/utils"
 	"github.com/samber/lo"
-	"github.com/stefanhaller/git-todo-parser/todo"
+	"github.com/fsmiamoto/git-todo-parser/todo"
 )
 
 // after selecting the 200th commit, we'll load in all the rest
--- a/pkg/gui/presentation/commits.go
+++ b/pkg/gui/presentation/commits.go
@@ -18,7 +18,7 @@
 	"github.com/kyokomi/emoji/v2"
 	"github.com/samber/lo"
 	"github.com/sasha-s/go-deadlock"
-	"github.com/stefanhaller/git-todo-parser/todo"
+	"github.com/fsmiamoto/git-todo-parser/todo"
 )
 
 type pipeSetCacheKey struct {
--- a/pkg/gui/presentation/commits_test.go
+++ b/pkg/gui/presentation/commits_test.go
@@ -13,7 +13,7 @@
 	"github.com/jesseduffield/lazygit/pkg/common"
 	"github.com/jesseduffield/lazygit/pkg/utils"
 	"github.com/samber/lo"
-	"github.com/stefanhaller/git-todo-parser/todo"
+	"github.com/fsmiamoto/git-todo-parser/todo"
 	"github.com/stretchr/testify/assert"
 	"github.com/xo/terminfo"
 )
--- a/pkg/gui/services/custom_commands/models.go
+++ b/pkg/gui/services/custom_commands/models.go
@@ -2,7 +2,7 @@
 
 import (
 	"github.com/jesseduffield/lazygit/pkg/commands/models"
-	"github.com/stefanhaller/git-todo-parser/todo"
+	"github.com/fsmiamoto/git-todo-parser/todo"
 )
 
 // We create shims for all the model classes in order to get a more stable API
--- a/pkg/utils/rebase_todo.go
+++ b/pkg/utils/rebase_todo.go
@@ -8,7 +8,7 @@
 	"slices"
 
 	"github.com/samber/lo"
-	"github.com/stefanhaller/git-todo-parser/todo"
+	"github.com/fsmiamoto/git-todo-parser/todo"
 )
 
 type Todo struct {
--- a/pkg/utils/rebase_todo_test.go
+++ b/pkg/utils/rebase_todo_test.go
@@ -5,7 +5,7 @@
 	"fmt"
 	"testing"
 
-	"github.com/stefanhaller/git-todo-parser/todo"
+	"github.com/fsmiamoto/git-todo-parser/todo"
 	"github.com/stretchr/testify/assert"
 )
 
